src/psim/simulations/single_orbit.cpp

```diff
@@ -41,11 +41,13 @@ SingleOrbitGnc::SingleOrbitGnc(Configuration const &config) {
   // Time and Earth ephemeris
   add<Time>(config, "truth");
   add<EarthGnc>(config, "truth");
-  // Orbital dynamics and lazy transformations
+  // Orbital dynamics
   add<OrbitGncEci>(config, "truth", "leader");
   add<TransformPositionEci>(config, "truth", "truth.leader.orbit.r");
   add<TransformVelocityEci>(config, "truth", "leader", "truth.leader.orbit.v");
-  // Environmental variables with body frame independent lazy transformations
+  // Environmental models
   add<EnvironmentGnc>(config, "truth", "leader");
+  add<TransformPositionEcef>(config, "truth", "truth.leader.environment.b");
+  add<TransformPositionEci>(config, "truth", "truth.leader.environment.s");
```

Comment:
These two lines here make the following transformations:

 * `truth.leader.environment.b` into `truth.eader.environment.b.eci` and `truth.eader.environment.b.ecef`
 * `truth.eader.environment.s` into `truth.eader.environment.s.eci` and `truth.eader.environment.s.ecef`